pkg/render/namespaces.go

```diff
@@ -108,7 +108,16 @@ func CreateNamespace(name string, provider operatorv1.Provider, pss PodSecurityS
 		ns.Annotations["security.openshift.io/scc.podSecurityLabelSync"] = "false"
 		ns.Labels["openshift.io/run-level"] = "0"
 	case operatorv1.ProviderAKS:
-		ns.Labels["control-plane"] = "true"
+		if applyAzurePolicy(azure, pss) {
+			ns.Labels["control-plane"] = "true"
+		}
 	}
 	return ns
 }
+
+func applyAzurePolicy(azure *operatorv1.Azure, pss PodSecurityStandard) bool {
+	if azure == nil || azure.PolicyMode == nil || *azure.PolicyMode == operatorv1.Default {
```

Comment:
I think we should only possibly return true if the PolicyMode is Default.

Reply:
Is it related to the previous comment, return "true" instead of  "PSSPrivileged == pss" or not to return true for nil cases ?

Reply:
I believe not having this config set (nil cases) should apply default behavior.

Reply:
It should, but it should also always default `nil` -> `Default`

Reply:
I think the tests in the core_controller and the renderer reflect the desired outcomes for each permutation and the tests pass.
So I think it is ok to proceed with merging this PR.